apps/recnet-api/src/database/repository/rec.repository.ts

```diff
@@ -128,9 +128,16 @@ export default class RecRepository {
     if (filter.userIds) {
       where.userId = { in: filter.userIds };
     }
-    if (filter.cutoff) {
+
+    if (filter.cutoff instanceof Date) {
       where.cutoff = filter.cutoff;
+    } else if (filter.cutoff) {
+      where.cutoff = {
+        gte: filter.cutoff.from || undefined,
+        lte: filter.cutoff.to || undefined,
+      };
```

Comment:
```suggestion
    } else if (filter.cutoff) {
      where.cutoff = {
        gte: filter.cutoff.from,
        lte: filter.cutoff.to,
      };
```

Nit: We can remove the "null-undefined" conversion here if we modify the type at `rec.repository.type.ts` to 
```ts
export type DateRange = {
  from?: Date;
  to?: Date;
};
```
and remove the `from: null` at `rec.service.ts`